app/api/v1/users.rb

```diff
@@ -0,0 +1,17 @@
+class V1::Users < Grape::API
+  before do
+    authenticate!
+  end
+
+  desc 'Return all users', { tags: ['user'] }
+  params do
+    optional :created_at, type: DateTime
+  end
+  paginate
+  get '/users/' do
+    users = User.all
+    users = users.where("created_at > ?", params[:created_at]) if params[:created_at]
```

Comment:
And I have another issue here. I don't remember how tokens were implemented, but this kind of endpoint downloads potentially every data from the users table. It's best to add a Pundit class to scope it out to just users that the User who issued the token has visibility to, or at least restricted to the team of the authorization. cc @andrerpbts